examples/aiohttp/README.md

```diff
@@ -0,0 +1,17 @@
+# aiohttp example
+This example demonstrates the integration with the [aiohttp](https://flask.palletsprojects.com/).
+
+## Running the example
+1. Start an instance of SBA (Spring Boot Admin):
+    ```sh
+    docker run -p 8082:8082 --rm michayaak/spring-boot-admin:2.2.2
+    ```
+2. Once Spring Boot Admin is running, you can run the examples as follow:
+    ```sh
+    cd examples/aiohttp
+    poetry install
+    poetry run python -m aiohttp_example_app
```

Comment:
I tried running this example following the instructions, and got the following error:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/matan.r/Code/SolarEdge/pyctuator/examples/aiohttp/aiohttp_example_app.py", line 31, in <module>
    app_description="Demonstrate Spring Boot Admin Integration with aiohttp",
  File "/Users/matan.r/Code/SolarEdge/pyctuator/examples/aiohttp/.venv/lib/python3.7/site-packages/pyctuator/pyctuator.py", line 122, in __init__
    raise EnvironmentError("No framework was found that is matching the target app"
OSError: No framework was found that is matching the target app(is it properly installed and imported?)
```

Reply:
I looked at the code and it seems like it should work. Does it work on your setup?

Reply:
Yes, its working for me.
Dod you:
```
poetry install --extras aiohttp
```

Reply:
I did not, and why should I? This is an aiohttp example, it doesn't make sense for aiohttp to be an optional dependency

Reply:
So the issue is that the example depends on a version of pyctuator that wasn't yet released...

To workaround this, you can use https://python-poetry.org/docs/dependency-specification/#path-dependencies, so you change the dependency in `pyproject.toml` to:
```toml
pyctuator = { path = "../../" }
```